features/step_definitions/organization_steps.rb

```diff
@@ -10,14 +10,34 @@
   add_named_record(name, create(:organization, options))
 end
 
-Given /user (\w+) who is a member of (.+) exists/ do |user_name,organization_name|
+Given /user (\w+)(?: with primary email (.+?))? who is a member of (.+) exists( and is inactive)?/ do |user_name,primary_email,organization_name,inactive|
```

Comment:
I think the tests would be a bit more readable if you moved the "with primary email" after the "who is a member of" so that it reads: "Given user foo who is a member of org exists with primary email foo@example.com and is inactive".

I suggest this because the primary information I want to see is the username and the organization, and the rest is additional information about that user.